docs/product/components/tables.html

```diff
@@ -631,6 +633,76 @@
             </table>
         </div>
     </div>
+
+    {% header h3 | JavaScript sorting %}
```

Comment:
At some point, I'd revisit the documentation for these components and make the API clearer. The step-by-step guidance is nice, but it'd be really nice to see something like a table with the available `data-` attributes, appropriate values, and what elements to apply them to in a component. I could glance at it quickly and go.

Not a blocker for getting JS out, but something to consider for the future.

Reply:
Huge 👍 on this one. This Tables page could use both an available classes table, as well as a new JS data attributes table. On it!